src/common/reset.css

```diff
@@ -32,3 +32,11 @@ ul {
 address {
   font-style: normal;
 }
+
+.visuallyhidden {
```

Comment:
Shouldn't this be camelCase? Looks weird as a single word

Reply:
👍